lambda/index.ts

```diff
@@ -65,21 +65,21 @@
 export const handler = function (
   event: Event<ResourceProperties>,
   context: Context,
-  callback: Callback
+  callback: Callback,
 ) {
   new CustomResource<ResourceProperties>(
     event,
     context,
     callback,
     createResource,
     updateResource,
-    deleteResource
+    deleteResource,
   );
 };
 
 function createResource(
   resource: CustomResource<ResourceProperties>,
-  log: Logger
+  log: Logger,
 ): Promise<void> {
   log.debug('called function createResource');
   log.info(`Attempting to create EC2 Key Pair ${resource.properties.Name}`);
```

Comment:
you are absolutely correct and i would love to change it. though in this case it's not possible. the interface describes the properties of a custom cloudformation resource and translate to the values Lambda receives in `event.ResourceProperties` and `event.OldResourceProperties`.

since this construct is already in use this would mess with property comparision.

let's assume I change the interface and the actual properties to camelCase, then on the first reousrce update `event.OldResourceProperties` would still contain the PascalCase keys and we would require some advanced logic to compare the keys to detect if an update is required.

Tha package aws-cloudformation-custom-resource already has comparision logic baked in where we can just do a `if(resource.Name.changed)` to compare `event.ResourceProperties.Name` and `event.OldResourceProperties.Name`, which I would like to keep.